src/services/api/AxiosInstance.ts

```diff
@@ -25,9 +27,11 @@ const requestFormatter = async (axiosConfig: AxiosRequestConfig) => {
     authMessage === "token undefined"
 
   if (isEmailLoginUser) {
-    const accessToken = await getAccessToken()
-    axiosConfig.headers = {
-      Authorization: `token ${accessToken}`,
+    const accessToken = await tokenServiceInstance
+      .getAccessToken()
+      .unwrapOr("null")
```

Comment:
Hmm I think we shouldnt use null at all, what do you feel about just resolving it to ""

Reply:
I am not too sure how to handle this error case. Currently leaning towards letting the request be made without throwing an error. Perhaps `"No Available Token"` will be more verbose?

Reply:
Hmm is there a reason why you are not inclined to throw/log an error here and exit the function immediately? 
This does not seem to me that this the below codes should not be run if we dont get an access code, wdyt

Reply:
My previous concerns were that cancelling the request at interceptor might cause unintended behaviour. As per Harish comments, it has been change to only make the request if `isOk`.